backend/core/src/main/java/org/sonarsource/sonarlint/core/tracking/TaintVulnerabilityTrackingService.java

```diff
@@ -202,13 +216,20 @@ private List<TaintVulnerabilityDto> loadTaintVulnerabilities(String configuratio
         }
         var projectStorage = storageService.binding(binding);
         var newCodeDefinition = projectStorage.newCodeDefinition().read().<Predicate<Instant>>map(definition -> definition::isOnNewCode).orElse(date -> true);
+        var isMQRMode = severityModeService.isMQRModeForConnection(binding.getConnectionId());
         var pathTranslationOpt = pathTranslationService.getOrComputePathTranslation(configurationScopeId);
         return pathTranslationOpt.map(translation -> projectStorage.findings().loadTaint(matchedBranch)
-          .stream().map(serverTaintIssue -> toDto(serverTaintIssue, newCodeDefinition, translation)).collect(Collectors.toList())).orElse(null);
+          .stream().map(serverTaintIssue -> toDto(serverTaintIssue, newCodeDefinition, translation, isMQRMode)).collect(Collectors.toList())).orElse(null);
       }).orElseGet(Collections::emptyList);
   }
 
-  private static TaintVulnerabilityDto toDto(ServerTaintIssue serverTaintIssue, Predicate<Instant> isOnNewCode, FilePathTranslation translation) {
+  private static TaintVulnerabilityDto toDto(ServerTaintIssue serverTaintIssue, Predicate<Instant> isOnNewCode,
+    FilePathTranslation translation, boolean isMQRMode) {
+    var cca = serverTaintIssue.getCleanCodeAttribute().map(attribute -> CleanCodeAttribute.valueOf(attribute.name())).orElse(null);
```

Comment:
Nitpick: More of a personal taste but I don't like abbreviations, and I think here you can afford a few extra characters